spec/factories.rb

```diff
@@ -11,6 +11,7 @@
     sequence(:resource_owner_id) { |n| n }
     application
     expires_in 2.hours
+    previous_refresh_token "token"
```

Comment:
I'd personally remove this line from the factory. First tokens don't have previous tokens, and it makes more sense to me as default state.